create-instance.sh

```diff
@@ -54,6 +54,72 @@ set_up_security_group() {
 
 }
 
+create_instance_profile() {
```

Comment:
it is good to have a separate script to create instance profile, we can call the script here to create profile.